src/components/organisms/Navbar/NavbarLinksList/NavbarLinksList.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { Fragment } from 'react';
+
+import NavbarDropdown, { ItemProps } from '../NavbarDropdown/NavbarDropdown';
+import { NavigationItem, NavbarLinksListProps } from '../Navbar/Navbar';
+
+export interface RenderItemProps {
+  item: NavigationItem;
+  getItemProps: () => ItemProps;
+}
+
+const NavbarLinksList: React.FC<NavbarLinksListProps> = ({ links, renderLink }) => (
+  <>
+    {links &&
+      links.map((item: NavigationItem, index: number) =>
+        !!item.children ? (
+          <NavbarDropdown
+            key={`dropdown-${index}`}
+            id={`navbar-dropdown-${index}`}
+            label={item.label}
+            items={item.children!}
+            renderItem={({ item, getItemProps }: RenderItemProps) =>
+              renderLink &&
+              renderLink(item, index, {
+                ...getItemProps(),
+                isDropdownLink: true,
+                'data-automation': item.qadata ?? `ZA.${item.qadata}`,
```

Comment:
We could probably avoid mapping `qadata` to `data-automation`, or even just spread the props